### ISSUE TITLE
Livro usado no exercícios

### ISSUE THREAD

**Issue by eltonrobaina** [vlsantos-bit/Exemplos_radar_livro-1#1]
Boa noite! Poderia informar qual livro está usando? Venho vendo seus códigos e eles tem sido de grande ajuda, se puder compartilhar o livro que está seguindo eu agradeço.

**Comment by vlsantos-bit** [comment]
Legal saber q está lhe ajudando, não estou seguindo livro nenhum hahah,eu
desenvolvo os códigos me baseando no material disponibilizado pelas
bibliotecas, mas não em um livro.

**Comment by eltonrobaina** [comment]
Que irado cara, muito bom mesmo. Eu add você no Linkdin, se possível gostaria da sua ajuda com a minha pesquisa. Sou estudante de mestrado e gostaria de utilizar dados de Radares Meteorológicos para prevenção de catástrofes na área de Engenharia da Defesa.  Gostaria de saber se teria disponibilidade para fazermos um meet e compartilhamos informações.

**Comment by vlsantos-bit** [comment]
Sim, eu vi e lhe respondi lá. Eu ando com a agenda um pouco cheia, mas
podemos conversar sim, realizo alguns freelancer dependendo do que seja.

Em dom., 2 de mai. de 2021 às 19:30, Elton John Robaina da Silva <
***@***.***> escreveu:

**Comment by eltonrobaina** [comment]
Positivo, vou continuar a conversa por lá!
